exporter/clickhouselogsexporter/migrations/000001_init_db.down.sql

```diff
@@ -4,4 +4,5 @@ DROP TABLE IF EXISTS signoz_logs.logs_resource_keys ON CLUSTER cluster;
 DROP TABLE IF EXISTS signoz_logs.resource_keys_string_final_mv ON CLUSTER cluster;
 DROP TABLE IF EXISTS signoz_logs.atrribute_keys_float64_final_mv ON CLUSTER cluster;
 DROP TABLE IF EXISTS signoz_logs.atrribute_keys_int64_final_mv ON CLUSTER cluster;
-DROP TABLE IF EXISTS signoz_logs.atrribute_keys_string_final_mv ON CLUSTER cluster;
\ No newline at end of file
+DROP TABLE IF EXISTS signoz_logs.atrribute_keys_string_final_mv ON CLUSTER cluster;
+DROP DATABASE signoz_logs ON CLUSTER cluster;
```

Comment:
We don't change the existing migrations. It changes the hash and leaves the state to be dirty?

Reply:
> I would think about the broader effects and changes needed. There are also migrations for traces, and there is no migrations directory for metrics for legacy reasons.

It's definitely a broader issue to tackle- hopefully, we can do one at a time and make sure each change is backwards-compatible. I have looked only at the logs exporter and looks like it only needs these permissions for the creation of DB, tables during startup.

> We don't change the existing migrations. It changes the hash and leaves the state to be dirty?

Oh, my bad. I've only tried on a clean installation. So to remain backward-compatible, would creating a dedicated directory work?

Reply:
>It's definitely a broader issue to tackle- hopefully, we can do one at a time and make sure each change is backwards-compatible.

For such broader issues, it's better to have a plan and list the tasks before starting to work on anything. This will give us a chance to think about and evaluate it. Whether or not we make it backwards-compatible or to what extent it will be backwards-compatible, we can take a call but I believe this should be discussed and evaluated before we implement a solution.

>So to remain backward-compatible, would creating a dedicated directory work?

No, a new directory doesn't change anything. The version number and content of each migration file are important. The version can be any unsigned integer. I do not see a version `0` so we could move the creation of the database to something like `000000_create_db.{up/down}.sql`.

Reply:
Absolutely, what's the best way for me to be involved in discussing this? Are there Signoz community calls, etc?

Reply:
Can you create an issue over here https://github.com/SigNoz/signoz. As I understand, the main ask is that RBAC-controlled ClickHouse should work with SigNoz. Along with the migrations in the collector, we have DDL manipulation in the main repo for enabling the TTL setting from the UI.

Reply:
Done! https://github.com/SigNoz/signoz/issues/2556

Reply:
>  I do not see a version 0 so we could move the creation of the database to something like 000000_create_db.{up/down}.sql.

Did this, and it works now on existing and clean installs. :D